ansible/configs/studentvm/files/repos_template.j2

```diff
@@ -0,0 +1,25 @@
+# RHEL 8 Repos
```

Comment:
If you intend on using Satellite with content views, this file is not even needed. It was only needed for bypassing that and accessing directly with certs. Now that we randomize names, there is really no good reason to do that...

Reply:
Let me try to delete and run one again.